[2023-01-13T08:48:09.087+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.is_api_avaible scheduled__2023-01-12T00:00:00+00:00 [queued]>
[2023-01-13T08:48:09.107+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.is_api_avaible scheduled__2023-01-12T00:00:00+00:00 [queued]>
[2023-01-13T08:48:09.108+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2023-01-13T08:48:09.108+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2023-01-13T08:48:09.108+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2023-01-13T08:48:09.303+0000] {taskinstance.py:1383} INFO - Executing <Task(HttpSensor): is_api_avaible> on 2023-01-12 00:00:00+00:00
[2023-01-13T08:48:09.331+0000] {standard_task_runner.py:55} INFO - Started process 163 to run task
[2023-01-13T08:48:09.343+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'user_processing', 'is_api_avaible', 'scheduled__2023-01-12T00:00:00+00:00', '--job-id', '95', '--raw', '--subdir', 'DAGS_FOLDER/user_processing.py', '--cfg-path', '/tmp/tmpm6k8yhxc']
[2023-01-13T08:48:09.345+0000] {standard_task_runner.py:83} INFO - Job 95: Subtask is_api_avaible
[2023-01-13T08:48:09.498+0000] {task_command.py:376} INFO - Running <TaskInstance: user_processing.is_api_avaible scheduled__2023-01-12T00:00:00+00:00 [running]> on host 274e1da73cdf
[2023-01-13T08:48:09.689+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=user_processing
AIRFLOW_CTX_TASK_ID=is_api_avaible
AIRFLOW_CTX_EXECUTION_DATE=2023-01-12T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-01-12T00:00:00+00:00
[2023-01-13T08:48:09.690+0000] {http.py:120} INFO - Poking: api/
[2023-01-13T08:48:09.708+0000] {base.py:71} INFO - Using connection ID 'user_api' for task execution.
[2023-01-13T08:48:09.710+0000] {http.py:148} INFO - Sending 'GET' to url: https://randomuser.me/api/
[2023-01-13T08:48:10.162+0000] {base.py:213} INFO - Success criteria met. Exiting.
[2023-01-13T08:48:10.201+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=user_processing, task_id=is_api_avaible, execution_date=20230112T000000, start_date=20230113T084809, end_date=20230113T084810
[2023-01-13T08:48:10.412+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2023-01-13T08:48:10.502+0000] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
